.gitlab-ci.yml

```diff
@@ -0,0 +1,12 @@
+stages:
+  - build
+
+.build-image:
+  image: markhobson/maven-chrome:jdk-16
+
+Build:
+  stage: build
+  extends:
+    - .build-image
+  script:
+    - mvn clean install
```

Comment:
Since you don`t cache sript results it is not necessarily to use install. Compile is enough.